### PR TITLE
core-cards

### PR DESCRIPTION
**Pull request by bigmoves**

### CODE REVIEW

**Comment by bigmoves** [comment]
@evrowe I agree. Cards are pretty boring but I'm sure we can spice them up a bit.

File: addon/components/core-card/component.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ *   {{#components.title}}Card title{{/components.title}}
+ *   {{#components.body}}Card body{{/components.body}}
+ *   {{#components.footer}}Card footer{{/components.footer}}
+ * {{/core-card}}
+ * ```
+ * @class Component.CoreCard
+ * @constructor
+ * @extends Ember.Component
+ */
+export default Ember.Component.extend({
```
Comment (evrowe) [inline addon/components/core-card/component.js]: Gotta switch this dude to `export default Component.extend`; the tests blow up otherwise :/

**Comment by evrowe** [comment]
Also, I think you should merge develop in and re-push; Dan had a failing test in develop b/c of an ES6 method (includes) that isn't supported in Node 4, and develop has a fix for that. Even once your tests are passing, the suite will fail until you have that in. Blerp.

**Comment by bigmoves** [comment]
@evrowe ok fixing now.

**Comment by evrowe** [comment]
YEAH YEAH YEAH

**Pull request merged**